### PR TITLE
[MI-2867]: Add search label auto-suggest hints and other style fixes

### PR DESCRIPTION
**Pull request by SaurabhSharma-884**
### Ticket ID:
https://brightscout.atlassian.net/browse/MI-2867

### Description:
- Add search label auto-suggest hints
- Add className prop and iconBgColor in NoData
- Centre Align text in NoData
- Fix background colour in Tabs
- Fix Option List Background in MMSearch
- Fix MMSearch option list all options change to from:

### Screenshots:
Screenshot from 2023-03-14 17-26-16
Screenshot from 2023-03-14 17-26-46

### CODE REVIEW

File: src/components/List/List.component.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import React, {forwardRef} from 'react';
+import React, { forwardRef } from 'react';
 
-import {MenuItem} from '@Components/MenuItem';
-import {extendClassname} from '@Utils';
+import { MenuItem } from '@Components/MenuItem';
```
Comment (Kshitij-Katiyar) [inline src/components/List/List.component.tsx:3]: why these changes ?

File: src/components/MMSearch/MMSearch.component.tsx [hunk 2]
```diff
@@ -163,6 +165,13 @@ export const MMSearch = (props: MMSearchProps) => {
                 label={label}
                 iconName={leadingIcon}
                 onClose={() => {
+                    if (onClearInput) {
+                        onClearInput();
+                    }
+                    if (inputRef.current) {
```
Comment (Kshitij-Katiyar) [inline src/components/MMSearch/MMSearch.component.tsx:171]: if there need to check whether inputRef is null or not ?

Comment (SaurabhSharma-884) [reply src/components/MMSearch/MMSearch.component.tsx:171]: yes, inputRef may be undefined and throws error if not checked

**Review by raghavaggarwal2308 — APPROVED** [review]

File: src/components/List/List.component.tsx (hunk 1 above)
Comment (SaurabhSharma-884) [reply src/components/List/List.component.tsx]: Fixed

**Review by Kshitij-Katiyar — APPROVED** [review]

File: src/components/MMSearch/MMSearch.component.tsx (hunk 2 above)
Comment (avas27JTG) [inline src/components/MMSearch/MMSearch.component.tsx:171]:
```suggestion
                    if (inputRef?.current) {
```

**Review by avas27JTG — APPROVED** [review]
